### PR TITLE
Update foreman_azure to 1.1.1 [deb]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.13
* [x] 1.12
* [x] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Thanks!

### CODE REVIEW

File: plugins/ruby-foreman-azure/debian/gem.list [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_azure-1.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_azure-1.1.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/fog-azure-0.0.2.gem"
 GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.5.gem"
```
Comment (domcleal) [inline plugins/ruby-foreman-azure/debian/gem.list]: Would you mind updating this to 0.7.7? It should fix the error seen in #1368 (which may recur here) and to remove the uuid/macaddr gems.

File: plugins/ruby-foreman-azure/debian/gem.list [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.1.gem"
```
Comment (domcleal) [inline plugins/ruby-foreman-azure/debian/gem.list]: and 0.1.5

**Comment by dLobatog** [comment]
Updated so that it uses 0.7.7 azure and 0.1.5 azure-core - thanks for the review @domcleal , I forgot about the old PR after so long

File: plugins/ruby-foreman-azure/debian/gem.list [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-GEMS="https://rubygems.org/downloads/foreman_azure-1.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_azure-1.1.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/fog-azure-0.0.2.gem"
-GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.5.gem"
-GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.1.gem"
+GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.7.gem"
+GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.5.gem"
 GEMS="$GEMS https://rubygems.org/downloads/faraday_middleware-0.10.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/systemu-2.6.5.gem"
 GEMS="$GEMS https://rubygems.org/downloads/uuid-2.3.8.gem"
```
Comment (domcleal) [inline plugins/ruby-foreman-azure/debian/gem.list]: uuid/macaddr should be removed.

File: plugins/ruby-foreman-azure/debian/gem.list [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_azure-1.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_azure-1.1.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/fog-azure-0.0.2.gem"
-GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.5.gem"
-GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.1.gem"
+GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.7.gem"
+GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.5.gem"
 GEMS="$GEMS https://rubygems.org/downloads/faraday_middleware-0.10.0.gem"
```
Comment (domcleal) [inline plugins/ruby-foreman-azure/debian/gem.list]: Updating faraday_middleware to 0.10.1 should permit faraday 0.10.0, fixing the build failure.

**Comment by mmoll** [comment]
it seems there's a similar issue regarding faraday now

**Comment by dLobatog** [comment]
Updated to 1.1.1 now  which works in 1.12 and 1.11. I've also updated the faraday version.

File: plugins/ruby-foreman-azure/debian/changelog [hunk 5]
```diff
@@ -1,3 +1,15 @@
+ruby-foreman-azure (1.1.1-1) stable; urgency=low
+
+  * 1.1.1 released
+
+ -- Daniel Lobato Garcia <me@daniellobato.me>  Tue, 15 Nov 2016 11:32:48 +0100
+
+ruby-foreman-azure (1.1.0-1) stable; urgency=low
```
Comment (domcleal) [inline plugins/ruby-foreman-azure/debian/changelog]: Nitpick: this can be removed, only 1.1.1's being built.

**Comment by dLobatog** [comment]
Updated - thanks, it was a leftover since I just updated the plugin by using the script

**Comment by domcleal** [comment]
Merged and building, thanks @dLobatog.

**Pull request merged**